### PR TITLE
Add IIS startup performance scenario

### PR DESCRIPTION
**Pull request by troydai**

### CODE REVIEW

File: test/Microsoft.AspNet.Tests.Performance/IISStartup.cs [hunk 1]
```diff
@@ -0,0 +1,49 @@
+﻿#if DNX451
```
Comment (NTaylorMullen) [inline test/Microsoft.AspNet.Tests.Performance/IISStartup.cs]: Header

File: test/Microsoft.AspNet.Tests.Performance/IISTestManager.cs [hunk 2]
```diff
@@ -0,0 +1,66 @@
+﻿using System;
```
Comment (NTaylorMullen) [inline test/Microsoft.AspNet.Tests.Performance/IISTestManager.cs]: ditto

File: test/Microsoft.AspNet.Tests.Performance/IISTestManager.cs [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+using System.Collections.Generic;
+using Benchmarks.Utility.Helpers;
+using Microsoft.AspNet.Server.Testing;
+using Microsoft.Extensions.Logging;
+
+#if DNX451
```
Comment (NTaylorMullen) [inline test/Microsoft.AspNet.Tests.Performance/IISTestManager.cs]: Move to top

File: testapp/StarterMvc/Startup.cs [hunk 4]
```diff
@@ -72,16 +72,16 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
...
-                    .CreateScope())
-                {
-                    var dbContext = serviceScope.ServiceProvider.GetService<ApplicationDbContext>();
-
-                    // We delete the pre-existing database first because each start of the Mvc site should be like running an app for the first time.
-                    dbContext.Database.EnsureDeleted();
-
-                    dbContext.Database.Migrate();
-                }
+                // using (var serviceScope = app.ApplicationServices.GetRequiredService<IServiceScopeFactory>()
```
Comment (NTaylorMullen) [inline testapp/StarterMvc/Startup.cs]: ? This'll break the StarterMvc stress test.

**Comment by NTaylorMullen** [comment]
Hmmm, getting to the point where we could significantly re-use the stress bits. WIll stop by.

File: testapp/StarterMvc/Startup.cs (hunk 4 above)
Comment (troydai) [inline testapp/StarterMvc/Startup.cs]: I'd like not to share the sample with stress test.

**Pull request merged**